src/patternfly/components/NotificationBadge/notification-badge.scss

```diff
@@ -0,0 +1,35 @@
+.pf-c-notification-badge {
+  --pf-c-notification-badge--after--Top: 0;
+  --pf-c-notification-badge--after--Right: 0;
+  --pf-c-notification-badge--after--Width: var(--pf-global--spacer--sm);
+  --pf-c-notification-badge--after--Height: var(--pf-global--spacer--sm);
+  --pf-c-notification-badge--after--BackgroundColor: var(--pf-global--active-color--400);
```

Comment:
should the default be `transparent` since you have to use a modifier? We also wouldn't need to update the default if we updated the "unread" color (or "read" color, if we introduce a color for that in the future) to match.